common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/PurchasedProductsFetcher.kt

```diff
@@ -15,19 +15,19 @@ import java.util.concurrent.TimeUnit
 class PurchasedProductsFetcher(
     private val deviceCache: DeviceCache,
     private val billing: BillingAbstract,
-    private val dateProvider: DateProvider = DefaultDateProvider()
+    private val dateProvider: DateProvider = DefaultDateProvider(),
```

Comment:
We haven't really been using trailing commas in this project. However, we've discussed this and we like them so we are planning to enable them for the project soon, so I think this is fine 👍